src/core/models/PrisonPopulationSnapshotMetric.ts

```diff
@@ -33,14 +34,51 @@ import { filterTimePeriod } from "./utils";
 export default class PrisonPopulationSnapshotMetric extends PathwaysMetric<PrisonPopulationSnapshotRecord> {
   accessor: keyof PrisonPopulationSnapshotRecord;
 
+  totalCount: number;
+
   constructor(
     props: BaseMetricConstructorOptions<PrisonPopulationSnapshotRecord> & {
       accessor: keyof PrisonPopulationSnapshotRecord;
     }
   ) {
     super(props);
     this.accessor = props.accessor;
+    this.totalCount = -1;
     this.download = this.download.bind(this);
+
+    reaction(
+      () => !this.isLoading,
+      () => {
+        this.totalCount = this.getTotalCount;
+      }
+    );
+  }
+
+  get getTotalCount(): number {
+    if (!this.allRecords) return this.totalCount;
+    const allRows = this.allRecords.filter(
+      (record: PrisonPopulationSnapshotRecord) => {
+        return (
+          // #TODO #1597 create tooling to reduce listing every dimension in filters
+          record.gender === "ALL" &&
+          record.legalStatus === "ALL" &&
+          record.facility === "ALL" &&
+          record.ageGroup === "ALL" &&
+          record.lengthOfStay === "ALL"
+        );
+      }
+    );
+
+    if (allRows.length === 0) {
+      throw new Error(`Metric ${this.id} is missing the "ALL" row`);
+    }
+
+    /* Since we are defaulting NULL dimension values to "ALL" when the
+     * record is created, we may end up with multiple rows where every
+     * dimension is "ALL". Since we rely on the "ALL" row for the total count,
+     * we will sort them based by count and take the largest.
+     */
+    return allRows.sort((a, b) => b.count - a.count)[0].count;
```

Comment:
Sorting the entire list is `O(n log n)`, but if you just want the largest element, you can map and take the max which is `O(n)`

```suggestion
    return Math.max(allRows.map(r => r.count))
```